Sources/Plasma/FeatureLib/pfPython/cyPythonInterface.cpp

```diff
@@ -1305,7 +1280,7 @@ int PythonInterface::getOutputAndReset(std::string *output)
         if (dbgOut != nullptr)
         {
             // then send it the new text
-            pyObjectRef retVal = plPython::CallObject(dbgOut, PyUnicode_FromStdString(strVal));
+            pyObjectRef retVal = plPython::CallObject(dbgOut, PyUnicode_FromSTString(strVal));
```

Comment:
Hmm, for some reason, just the `const ST::string&` overload isn't enough here. I also had to add one for non-const `ST::string&`, otherwise it went to the `delete`d generic overload.